importer/tests/test_import.py

```diff
@@ -7,15 +7,44 @@
 import os
 from os.path import abspath, dirname, join
 from tempfile import mkstemp
-from shutil import copyfile
+import uuid
 import zipfile
 
+from django.conf import settings
+from django.core.files.storage import default_storage
+from django.core.files.base import ContentFile
+
 from importer.api import import_course_from_file
 from importer.tasks import import_file
 from learningresources.models import LearningResource, Course
 from learningresources.tests.base import LoreTestCase
 
 
+def copy_file(path):
+    """
+    Copy given file into django default_storage
+
+    Args:
+        path (str): Path to file to copy
+
+    Returns:
+        path (str): default_storage path to copy
+    """
+    if path.endswith('.tar.gz'):
+        ext = '.tar.gz'
+    else:
+        _, ext = os.path.splitext(path)
+
+    return default_storage.save(
+        '{prefix}/{random}{ext}'.format(
+            prefix=settings.IMPORT_PATH_PREFIX,
+            random=str(uuid.uuid4()),
+            ext=ext
+        ),
+        open(path, 'rb')
```

Comment:
Well we get to be a bit lazy in tests since we are using input from our own repo and want uncaught exceptions if the file isn't there so the tests fail.